docs/strategy_reviewable.md

```diff
@@ -0,0 +1,255 @@
+# Gas Cost Estimator approach strategy
+
+Here we organize thoughts on different approaches to consider.
+We focus on EVM for simplicity, but this should be substitutable by ewasm (Ethereum flavored WebAssembly) throughout the document.
```

Comment:
I think it is better to just say "WebAssembly interpreter" not "ewasm". The "Ethereum flavored" does not modify anything on the opcode level.

And by "interpreter" I mean that the similar instrumentation may be applied as in EVM. For JITs / compiler this may not be some straightforward.